spec/models/keywords_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# frozen_string_literal = true
+
+RSpec.describe Keyword, type: :model do
+  describe '#model' do
```

Comment:
Referring to Compass, this is [grouping the instance method tests](https://nimblehq.co/compass/development/code-conventions/ruby/rspec/#convention-3) so should it be `#update_status`?

```suggestion
  describe '#update_status' do
```

Reply:
Updated here https://github.com/aetheryna/google-scraper-ruby/pull/36/commits/5ccdd75db2c408cb66f542abf903918ab83b1750 👍🏾